### DISCUSSION TITLE
How do I make the mod version in fabric.mod.json use the mod_version in gradle.properties?

### DISCUSSION THREAD

**Discussion by TechPro424** [FabricMC/fabric-api#2844] [Mod Dev Support] [answered]
How do I make the mod version in `fabric.mod.json` use the `mod_version` in `gradle.properties`?

**Comment by warjort** [comment, marked as answer]
https://github.com/FabricMC/fabric-example-mod/blob/9cfe28c5eab9cafdd1ffd24611989b04b7e1926a/build.gradle#L40

https://github.com/FabricMC/fabric-example-mod/blob/9cfe28c5eab9cafdd1ffd24611989b04b7e1926a/src/main/resources/fabric.mod.json#L4

**Reply by TechPro424** [reply]
`build.gradle`
![image](https://user-images.githubusercontent.com/86620085/212229956-da96131d-8f8a-4193-90e3-dcbc87bbfd6b.png)

`fabric.mod.json`
![image](https://user-images.githubusercontent.com/86620085/212229968-9b7ccbc1-d326-4207-9013-8f25837b4923.png)

I have both the same way as the Fabric Example Mod, however, in-game, my mod version shows up as `${version}`
![image](https://user-images.githubusercontent.com/86620085/212230263-bb8ab77d-3184-4b28-9a1a-4dc6ba7bb2c9.png)

**Reply by haykam821** [reply]
If you're using Visual Studio Code's launch configurations, `processResources` will not be applied. Is the version correct when using the `runClient` or `build` tasks?

**Reply by Sollace** [reply]
When you're debugging it through the IDE, the version will show up as ${version} but that's only because processResources is only performed during a build. The jar file resulting from `gradlew build` that you upload to modrinth/curseforge or give out to share will have the correct version.

**Reply by TechPro424** [reply]
> When you're debugging it through the IDE, the version will show up as ${version} but that's only because processResources is only performed during a build. The jar file resulting from `gradlew build` that you upload to modrinth/curseforge or give out to share will have the correct version.

Yep, it worked when I used `runClient`
